client/js/screens/Campaign/Campaign.js

```diff
@@ -15,16 +15,17 @@ import MainUnsubscribeButton from '../../components/MainUnsubscribeButton';
 import MrMoneyImage from '../../assets/artwork/mrmoney.png';
 
 const Campaign = ({navigation, user}) => {
+  const isSubscribed = navigation.state.params.campaign.subscribers.find(
+    subscriber => subscriber.id === user.id,
+  );
+
```

Comment:
I thought we changed this. Or is this a separate issue?

Reply:
separate issue